Traducir.Web.Net/Startup.cs

```diff
@@ -0,0 +1,70 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Traducir.Web.Net
+{
+    public class Startup
+    {
+        //TODO: incorporate the relevant code from this class and remove this field (and the linked class file) when the Api project is deleted
```

Comment:
yeah, let's do that :) We could have the Api have a reference to this project (and have the Api have a reference to this class instead... so that what's on this project is correct and the Api is in a weird state)

Reply:
I prefer to leave it as is, the Api project untouched and the new project referencing the Api files. This way 1) if something changes in the Api project while I'm working on this it'll be easier to merge the changes in my branch, and 2) I want to be able run the old and the new projects simultaneously, and for that it's convenient to keep the Api project in its original state.

Reply:
👍